platform-core-module/src/main/java/com/softicar/platform/core/module/component/external/license/rule/ExternalComponentLicensePredefinedRules.java

```diff
@@ -23,6 +23,11 @@ private ExternalComponentLicensePredefinedRules() {
 
 		this.list = new ArrayList<>();
 
+		// Artifact does not contain license information.
+		// License defined at:
+		// https://github.com/google/guava/blob/master/COPYING
```

Comment:
This seems to be wrong. `bcprov` is a "bouncycastle" library. AFAIK, that's completely non-associated with "guava".

Bouncycastle even have their own license; see https://www.bouncycastle.org/licence.html